README.md

```diff
@@ -691,9 +693,151 @@ As a challenge, add a button for creating these new posts. Place this link insid
 ![Submit form and submit button](images/4.png)
 
 ## 6.0 Displaying Items in Collections
-*working title*
+*Reference Code is in Blask-6*
+
+If your blog posts have longer bodies, you may want them to be displayed on their own page, separate from the list of all pages. Let's make a new route in `routes/blog.py` to give each post its own page.
+
+Before we get into the application logic, here are some additional styles to add to the bottom of `static/css/styles.css`. It is based loosely off of BEM (Block Element Modifier) CSS syntax.
```

Comment:
It seems like using traditional CSS selectors would reduce confusion (like `.post-img-thumb` instead of `.post__img-thumb`).  Do you think it's important to use BEM?